### PR TITLE
⚡️➡️ (amp onwards)

### PR DESCRIPTION
**Pull request by AWare**
start of onwards

## Moustache Templates!

So moustache templates use `{{curly-braces}}` everywhere, which are not quite impossible to represent as output strings in jsx.

Options: 
1:  strings and dangerous html
2: `{'{'}`
3:  components

So we now have 
- `<MoustacheSection name='hello'></MoustacheSection>` => `{{#hello}}{{/hello}}`
- `<MoustacheVariable name='hello'/>` => '{{hello}}'
- `moustacheVariable('hello')` => `{{hello}}` (we use a fair few {{'s in props

### CODE REVIEW

**Comment by nicl** [comment]
Hey, thanks for this. It would probably be helpful for you to include a bit on the mustache approach you've taken in the description as that's quite interesting and worth discussion.

I'll have a think about this tomorrow too. My immediate reaction is:

* the problem is real
* I think this helps but I'm not sure yet how much
* 'Section' is probably worth renaming as it is easy to confuse with the real html element 'section'

**Comment by AWare** [comment]
Yeah, `<Section>` is a terrible name for the component. I'm not sure what would be better, so I tooke the name from the moustache docs https://mustache.github.io/mustache.5.html

I might pull `<template>` out as well and see what I can do with the lot.

**Comment by nicl** [comment]
@AWare thanks for this! MoustacheVariable and MoustacheSection are definitely improvements. Let's see if we can come up with an even better naming scheme later on, but this is really great!

Minor query: can we have just one of moustacheVariable and MoustacheVariable? Unless there is a good reason for presenting two ways to do something, I always vote for just one even if it's a bit less flexible.

File: packages/frontend/amp/components/Body.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Elements } from '@frontend/amp/components/lib/Elements';
 import { css } from 'react-emotion';
 import { ArticleModel } from '@frontend/amp/pages/Article';
 import { MainBlock } from '@frontend/amp/components/MainBlock';
+import { Onwards } from './Onwards';
```
Comment (nicl) [inline packages/frontend/amp/components/Body.tsx]: minor @ ..

**Pull request merged**